src/Listener/BearerListener.php

```diff
@@ -36,7 +35,7 @@ public function __construct(
         $this->logger = $logger;
     }
 
-    public function handle(GetResponseEvent $event)
+    public function handle(GetResponseEvent $event): void
```

Comment:
GetResponseEvent is deprecated, use \Symfony\Component\HttpKernel\Event\KernelEvent instead, it's already available in sf 4